module-1/lab-api-scavenger-game/your-code/challenge-3.ipynb

```diff
@@ -10,7 +10,9 @@
   {
```

Comment:
Great approach!! You've found the hidden message!! You can optimize your code a little bit, jejje.